doc/nrf/installation/recommended_versions.rst

```diff
@@ -77,15 +77,13 @@ The table uses the following tier definitions to categorize the level of operati
      The specified architecture is not supported for the respective operating system.
 
 Linux-only Zephyr features
-==========================
+  There are some Zephyr features that are currently only available on Linux, including:
 
-There are some Zephyr features that are currently only available on Linux, including:
-
-* twister
-* BlueZ integration
-* net-tools integration
-* Native Port (native_posix)
-* BabbleSim
+  * twister
```

Comment:
@PerMac

Reply:
Twister is available on Windows and works.  Tests for native posix and qemu won't work, but it is not a twister issue, but the fact that does platforms are not supported  on Windows

Reply:
BlueZ and net-tools: no idea, 
no native posix on Windows -> correct
babblesim is based on posix -> correct

Reply:
As for mac:
twister in general works
In particular qemu and hw should work
native posix doesn't work, hence bsim as well